server/routeHandlers.go

```diff
@@ -53,103 +53,107 @@ func newGame(c echo.Context) error {
 }
 
 func login(c echo.Context) error {
-    player, joinErr := joinGame(c.Param("game"), c.Param("username"))
-    
-    if joinErr != nil {
+	isHost := false
+	if c.Param("isHost") == "true" {
```

Comment:
I'm concerned that this may allow any player to be the host.

Reply:
When they created the new game, they receive a JWT that says they are the host for the game they create. It would be best to use that JWT here instead of the c.Param.

Reply:
Good point. There will need to be more validation on the client and the back-end. This was an attempt to allow from some way to notify the server that a user can host. Hopefully it's not too pre-mature.